app/actions/views/select_team.js

```diff
@@ -35,7 +35,7 @@ export function handleTeamChange(teamId, selectChannel = true) {
             markChannelAsRead(lastChannelId, currentChannelId)(dispatch, getState);
         }
 
-        dispatch(batchActions(actions), getState);
+        dispatch(batchActions(actions, 'BATCH_SELECT_TEAM'), getState);
```

Comment:
Maybe we could just add another action to the batch instead of changing the type of the batched action? Or just have the middleware look for a regular SELECT_TEAM action instead of the whole thing? 0/5

Reply:
Oh, or does the middleware only see a BATCH action when you do that so you can't filter for the SELECT_TEAM itself?

Reply:
yup the middleware only sees `BATCH_ACTION` and not each individual action, that is why I changed the name